fictadvisor-web/src/components/pages/account-page/components/group-tab/GroupTab.tsx

```diff
@@ -31,11 +30,26 @@ const getStudents = async (user: User) => {
   };
 };
 
+enum OrderTypes {
+  ascending = 'asc',
+  descending = 'desc',
+}
+
```

Comment:
We already have ascending/descending enum for other pages. Use it.
Else, it should be in types of Group API.